deploy/role.yaml

```diff
@@ -32,49 +34,45 @@ rules:
   - list
   - watch
   - patch
-  - update
 - apiGroups:
   - extensions
   - apps
   resources:
   - deployments
   - replicasets
   - daemonsets
-  - statefulsets
   verbs:
-  - '*'
+  - create
+  - get
+  - list
+  - patch
+  - watch
 - apiGroups:
   - ""
   resources:
   - serviceaccounts
   - configmaps
-  - nodes
-  - endpoints
-  - events
-  - secrets
```

Comment:
yep, unneeded because we don't create our own secrets (for template-validator) anymore